viewing_party/party.py

```diff
@@ -1,23 +1,168 @@
+import copy
 # ------------- WAVE 1 --------------------
+movies = {}

```

Comment:
We should keep our variables as locally scoped as possible to avoid them potentially being altered in places we don't expect them to be. Since it is only used by the `create_movie` function we should declare `movies` inside of that function (inside our if-block so we don't allocate space for it unless it will be used).